backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadService.java

```diff
@@ -114,48 +126,67 @@ public PatientBulkUploadResponse processPersonCSV(InputStream csvStream, UUID fa
             extractedData.getFirstName().getValue(),
             extractedData.getLastName().getValue(),
             parseUserShortDate(extractedData.getDateOfBirth().getValue()),
-            org,
+            currentOrganization,
             facility)) {
           continue;
         }
 
-        _personService.addPatient(
-            facilityId,
-            null, // lookupID
-            extractedData.getFirstName().getValue(),
-            extractedData.getMiddleName().getValue(),
-            extractedData.getLastName().getValue(),
-            extractedData.getSuffix().getValue(),
-            parseUserShortDate(extractedData.getDateOfBirth().getValue()),
-            address,
-            country,
-            List.of(
-                new PhoneNumber(
-                    parsePhoneType(extractedData.getPhoneNumberType().getValue()),
-                    extractedData.getPhoneNumber().getValue())),
-            parsePersonRole(extractedData.getRole().getValue(), false),
-            List.of(extractedData.getEmail().getValue()),
-            convertRaceToDatabaseValue(extractedData.getRace().getValue()),
-            convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()),
-            null,
-            convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()),
-            parseYesNo(extractedData.getResidentCongregateSetting().getValue()),
-            parseYesNo(extractedData.getEmployedInHealthcare().getValue()),
-            null,
-            null);
+        // create new person with current organization, then add to new patients list
+        Person newPatient =
+            new Person(
+                currentOrganization,
+                null,
+                extractedData.getFirstName().getValue(),
+                extractedData.getMiddleName().getValue(),
+                extractedData.getLastName().getValue(),
+                extractedData.getSuffix().getValue(),
+                parseUserShortDate(extractedData.getDateOfBirth().getValue()),
+                address,
+                country,
+                parsePersonRole(extractedData.getRole().getValue(), false),
+                List.of(extractedData.getEmail().getValue()),
+                convertRaceToDatabaseValue(extractedData.getRace().getValue()),
+                convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()),
+                null,
+                convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()),
+                parseYesNo(extractedData.getResidentCongregateSetting().getValue()),
+                parseYesNo(extractedData.getEmployedInHealthcare().getValue()),
+                null,
+                null);
+        newPatient.setFacility(assignedFacility); // might be null, that's fine
+        patientsList.add(newPatient);
```

Comment:
Can we add the `newPatient` to the `patientsList` when we are done modifying it (line 182)? I think it makes it more clear that way.

Reply:
done!